apps/docs/src/app/core/component-docs/alert/alert-header/alert-header.component.html

```diff
@@ -1,4 +1,7 @@
 <header>Alert</header>
+<fd-alert [type]="'warning'">
```

Comment:
Should we use message strip for these warnings?

Reply:
Done